src/test/java/uk/gov/justice/probation/courtcaseservice/service/OffenderServiceTest.java

```diff
@@ -38,8 +38,8 @@ public void setUp() {
         service = new OffenderService(offenderRestClient);
         offender = Offender.builder().build();
         when(offenderRestClient.getOffenderByCrn(CRN)).thenReturn(Mono.just(offender));
-        when(offenderRestClient.getConvictionsByCrn(CRN)).thenReturn(Mono.just(convictions));
-        when(offenderRestClient.getConvictionRequirements(CRN, CONVICTION_ID)).thenReturn(Mono.just(requirements));
+        when(offenderRestClient.getConvictionsByCrn(CRN)).thenReturn(Mono.just(expectedConvictions));
```

Comment:
Would like to see these mocked control statements moved into just the test methods which use them. ...